docs/resources/tf_export.md

```diff
@@ -60,6 +60,7 @@ resource "genesyscloud_tf_export" "export" {
 
 ### Optional
 
+- `compress` (Boolean) Flag to check if compression is allowed Defaults to `false`.
```

Comment:
We should add a test case to test an export and then make sure it is zipped (e.g. with a non-zero zip file"